classes/controller/FrontController.php

```diff
@@ -1084,7 +1084,8 @@ public function registerStylesheet($id, $relativePath, $params = [])
                 . ($this->stylesheetManager->getFullPath($relativePath) ?? $relativePath);
             $params['server'] = 'remote';
         }
-        $this->stylesheetManager->register($id, $relativePath, $params['media'], $params['priority'], $params['inline'], $params['server'], true, $params['version']);
+
+        $this->stylesheetManager->register($id, $relativePath, $params['media'], $params['priority'], $params['inline'], $params['server'], isset($params['needRtl']) ? $params['needRtl'] : true, $params['version']);
```

Comment:
@samberrry maybe add `needRtl` to `$default_params`?

Reply:
done in new PR